pyaedt/edb_core/edb_data/primitives_data.py

```diff
@@ -715,6 +716,27 @@ def length(self):
                 length += GeometryOperators.points_distance(center_line[pt_ind], center_line[pt_ind + 1])
         return length
 
+    @pyaedt_function_handler
+    def add_point(self, x, y, incremental=False):
```

Comment:
This needs to have a description. It seems that this docstring should be the same as the one for ``def add_point`` in previous database.py file.